modules/importer/src/main/scala/ConfigReader.scala

```diff
@@ -37,26 +37,33 @@ object ConfigReader3 {
+        ufindp(f)(s.sequenceValue)
+      }
+
     def seq[S <: SequenceableSpType, A: Enumerated: TypeTag](f: A => String): Read[A] =
       cast[S].map(s => ufindp(f)(s.sequenceValue))
 
-    val unit:        Read[Unit       ] = _ => ()
-    val string:      Read[String     ] = cast[String]
-    val double:      Read[Double     ] = cast[Double]
-    val int:         Read[Int        ] = cast[Int]
-    val long:        Read[Long       ] = cast[Long]
-    val offsetAngle: Read[Angle      ] = string.map(s => Angle.fromArcsecs(s.toDouble))
-    val offsetP:     Read[OffsetP    ] = offsetAngle.map(OffsetP(_))
-    val offsetQ:     Read[OffsetQ    ] = offsetAngle.map(OffsetQ(_))
-    val instrument:  Read[Instrument ] = enum(_.tccValue)
-    val yesNo:       Read[Boolean    ] = cast[YesNoType].map(_.toBoolean)
-    val durSecs:     Read[Duration   ] = double.map(_.toInt).map(Duration.ofSeconds(_))
-    val gcalLamp:    Read[GCalLamp   ] = cast[JSet[Object]].map(_.iterator.next.toString).map(ufindp[GCalLamp, String](_.tccValue))
-    val gcalShutter: Read[GCalShutter] = seq[OldGCal.Shutter, GCalShutter](_.tccValue)
-    val f2fpu:       Read[F2FpUnit   ] = seq[OldF2.FPUnit,    F2FpUnit   ](_.tccValue)
-    val f2filter:    Read[F2Filter   ] = seq[OldF2.Filter,    F2Filter   ](_.tccValue)
-    val f2lyotwheel: Read[F2LyotWheel] = seq[OldF2.LyotWheel, F2LyotWheel](_.tccValue)
-    val f2disperser: Read[F2Disperser] = seq[OldF2.Disperser, F2Disperser](_.tccValue)
+    val unit:          Read[Unit         ] = _ => ()
+    val string:        Read[String       ] = cast[String]
+    val double:        Read[Double       ] = cast[Double]
+    val int:           Read[Int          ] = cast[Int]
+    val long:          Read[Long         ] = cast[Long]
+    val offsetAngle:   Read[Angle        ] = string.map(s => Angle.fromArcsecs(s.toDouble))
+    val offsetP:       Read[OffsetP      ] = offsetAngle.map(OffsetP(_))
+    val offsetQ:       Read[OffsetQ      ] = offsetAngle.map(OffsetQ(_))
+    val instrument:    Read[Instrument   ] = enum(_.tccValue)
+    val yesNo:         Read[Boolean      ] = cast[YesNoType].map(_.toBoolean)
+    val durSecs:       Read[Duration     ] = double.map(_.toInt).map(Duration.ofSeconds(_))
+    val gcalLamp:      Read[GCalLamp     ] = cast[JSet[Object]].map(_.iterator.next.toString).map(ufindp[GCalLamp, String](_.tccValue))
+    val gcalShutter:   Read[GCalShutter  ] = seq[OldGCal.Shutter, GCalShutter](_.tccValue)
+    val f2fpu:         Read[F2FpUnit     ] = seq[OldF2.FPUnit,      F2FpUnit     ](_.tccValue)
+    val f2filter:      Read[F2Filter     ] = seq[OldF2.Filter,      F2Filter     ](_.tccValue)
+    val f2lyotwheel:   Read[F2LyotWheel  ] = seq[OldF2.LyotWheel,   F2LyotWheel  ](_.tccValue)
+    val f2disperser:   Read[F2Disperser  ] = seq[OldF2.Disperser,   F2Disperser  ](_.tccValue)
+    val f2windowcover: Read[F2WindowCover] = seqOpt[OldF2.WindowCover, F2WindowCover](_.tccValue, OldF2.WindowCover.OPEN)
```

Comment:
Here I use `seqOpt` to convert the window cover.  It's vaguely coming back to me though that this may not be a good idea.  I think in the seqexec there is a difference between defining a value for an expert field like `WindowCover` and simply leaving it be ...